### ISSUE TITLE
Add universal resource recovery option for some systems

### ISSUE THREAD

**Issue by yalinli2** [QSD-Group/EXPOsan#27]
Add a module-wise `INCLUDE_RESOURCE_RECOVERY` option for `biogenic_refinery`, `new_generator`, and `reclaimer` with default values to `False` (or the baseline to the publication, if there's one)

this should adjust the price/CFs in `__init__.py`, and make sure the models would be adjust accordingly

**Comment by yalinli2** [comment]
@haclohman I've added this option for biogenic_refinery, new_generator, and reclaimer, I also updated the scripts in EXPOsan/EXPOsan-private/DMsan to reflect these changes. But I didn't do a sweeping check on if this option works as expected.

So I'll recommend at least running the DMsan [sys_simulation](https://github.com/QSD-Group/DMsan/blob/main/dmsan/comparison/sys_simulation.py) script to compare the results for with vs. without resource recovery.

I'll leave you to close this issue if things are good/follow up if these are bugs!
